code/modules/food_and_drinks/machinery/oven.dm

```diff
@@ -90,7 +90,11 @@
 		baked_item.fire_act(1000) //Hot hot hot!
 
 		if(SPT_PROB(10, seconds_per_tick))
-			visible_message(span_danger("You smell a burnt smell coming from [src]!"))
+			var/list/asomnia_hadders = list()
+			for(var/mob/smeller in get_hearers_in_view(DEFAULT_MESSAGE_RANGE, src))
```

Comment:
```suggestion
			for(var/mob/smeller as a ything in get_hearers_in_view(DEFAULT_MESSAGE_RANGE, src))
```

Reply:
`anything` is misspelled btw so be careful if you hit the funny github commit suggestion button